### PR TITLE
95 - Making cli more consistent

### PR DESCRIPTION
**Pull request by Pbarbosa92**
This PR aims to improve the consistency of our cli help menus.
Adding descriptions to every subcommand, adding help to every option, and making the text more consistent all across.

### CODE REVIEW

File: inductiva/_cli/cmd_resources/_start.py [hunk 1]
```diff
@@ -22,7 +22,9 @@ def start_machine_group(args):
 def register(parser):
     """Register the launch of resources commands."""
 
-    subparser = parser.add_parser("start", help="Start computational resources")
+    subparser = parser.add_parser("start",
```
Comment (AugustoPeres) [inline inductiva/_cli/cmd_resources/_start.py:25]: I though we had removed this: https://github.com/inductiva/inductiva/pull/1031

**Review by AugustoPeres — COMMENTED** [review]
Looks good, I just have a question related to the `inductiva resources start`. Additionally I just noticed that some `descriptions` have periods at the end and others do not. This should be consistent across the menus.

File: inductiva/_cli/cmd_resources/_start.py (hunk 1 above)
Comment (Pbarbosa92) [reply inductiva/_cli/cmd_resources/_start.py:25]: We did. But not like this. On the loader we ignore files or folders started with "_"

**Review by AugustoPeres — APPROVED** [review]
LGTM

**Review by IvanPombo — COMMENTED** [review]
There is one boring but important job for the future - the messages in the description and help are repeated. You know what to do :D 

You could either add it directly to the localization or create a `msg` that you write once and apply in both places.

**Comment by Pbarbosa92** [comment]
> once and ap

If we decide that we can reuse the same message for help and description, I can move them to the localization folder. :D

File: inductiva/_cli/cmd_resources/cost.py [hunk 2]
```diff
@@ -28,20 +28,21 @@ def register(parser):
 
     subparser = parser.add_parser(
         "cost",
-        help="Estimate cost of a machine in the cloud",
+        description="Estimate cost of a machine in the cloud.",
+        help="Estimate cost of a machine in the cloud.",
```
Comment (sergiomsantos) [inline inductiva/_cli/cmd_resources/cost.py:32]: help and description are the same?

see https://github.com/inductiva/inductiva/pull/1037#issuecomment-1924884876 on how it should be.
I'll open a task for this.

**Review by sergiomsantos — APPROVED** [review]

**Review by sergiomsantos — APPROVED** [review]

**Comment by Pbarbosa92** [comment]
**QA**

Test CLI overall. Since this task changed almost all cli commands we need to test them all.

**Comment by AugustoPeres** [comment]
Screenshot 2024-02-06 at 10 51 55

Screenshot 2024-02-06 at 10 53 48

**Comment by Pbarbosa92** [comment]
**QA**

Minor changes. All looks good.

**Pull request merged**